docker/aws/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM eclipse-temurin:17
+
+LABEL org.opencontainers.image.source=https://github.com/xtdb/xtdb
+LABEL org.opencontainers.image.description="XTDB 2.x"
+LABEL org.opencontainers.image.licenses="The MIT License"
+
+WORKDIR /usr/local/lib/xtdb
+
+ENTRYPOINT ["java", \
+    "-Dclojure.main.report=stderr", \
+    "-Dlogback.configurationFile=logback.xml", \
+    "--add-opens=java.base/java.nio=ALL-UNNAMED", \
+    "-Dio.netty.tryReflectionSetAccessible=true", \
+    "-cp","xtdb-aws.jar", \
+    "clojure.main", "-m", "xtdb.main"]
+
+HEALTHCHECK --start-period=15s --timeout=3s \
+    CMD curl -f http://localhost:3000/status || exit 1
+
+RUN mkdir -p /var/lib/xtdb
+VOLUME /var/lib/xtdb
+
+ARG GIT_SHA
+ARG XTDB_VERSION
+ENV GIT_SHA=${GIT_SHA}
+ENV XTDB_VERSION=${XTDB_VERSION}
+
+LABEL org.opencontainers.image.version=${XTDB_VERSION}
+LABEL org.opencontainers.image.revision=${GIT_SHA}
+
+ADD aws_config.edn xtdb.edn
```

Comment:
I think you can now do:

```docker
COPY <<EOF xtdb.edn
{:xtdb.kafka/log {:topic-name #env XTDB_TOPIC_NAME
                  :bootstrap-servers #env KAFKA_BOOTSTRAP_SERVERS} 
 :xtdb.s3/bucket {:bucket #env XTDB_S3_BUCKET,
                  :prefix "xtdb-object-store"
                  :sns-topic-arn #env XTDB_SNS_TOPIC}
 :xtdb/server {:port 3000}}
EOF
```

if you'd prefer to inline this file?

Reply:
As discussed - decided not to in-line it for now, but no strong preference - just a slight assumption that down the line if we set up configurable docker files that will probably be using external files